mentions/build.gradle.kts

```diff
@@ -54,6 +54,14 @@ kotlin {
     explicitApi()
 }
 
+/**
+ * Kotlin code compilation task waits until
+ * Protobuf files are fully generated and rejections are created.
+ */
+tasks.named("compileKotlin") {
```

Comment:
Please have an issue (to address much-much later), telling that it is not ideal to establish the explicit dependencies between Gradle tasks like that. And the recommended way of ordering the tasks should be used, i.e. by using task inputs and outputs.

After you create such an issue, please have it pasted as a link here under a `TODO` item.

There are some examples throughout Spine codebase (look for `TODO`s).

Reply:
@MykytaPimonovTD please read my comment again. 

I have asked to "paste it as a link", meaning the link to the issue on GitHub. 

Have you checked other Spine code snippets with `TODO` items, like I advised?